src/Core/Migration/V6_5/Migration1664541794AddIndexForTasks.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_5;
```

Comment:
Is there a reason you put it into 6.5 folder? This means the migration will only be executed with the 6.5 update, but not with the next minor releases.

Alternatively you can put it into the 6.4 folder, so it will be executed directly with the next release. But then you also need to add the migration in the default `Shopware\Core\Migration` namespace for BC reasons